pkgs/development/interpreters/luajit/default.nix

```diff
@@ -79,9 +81,9 @@ stdenv.mkDerivation rec {
     {
       echo -e '
         #undef  LUA_PATH_DEFAULT
-        #define LUA_PATH_DEFAULT "./share/lua/${luaversion}/?.lua;./?.lua;./?/init.lua"
+        #define LUA_PATH_DEFAULT "./?.lua;./share/lua/${luaversion}/?.lua;./share/lua/${luaversion}/?/init.lua;"
```

Comment:
this drops the `./?/init.lua`, is that intentional?

also, can you check if your issue is solved by #289135 ?

Reply:
see my first command where default luajit's lua_path doesn't contain it.
It can be seen here as well https://github.com/LuaJIT/LuaJIT/blob/0d313b243194a0b8d2399d8b549ca5a0ff234db5/src/luaconf.h#L23

Reply:
consulting with luajit devs, this is intentional, but actually `./?/init.lua` should *only* be present in lua 5.3

Reply:
interesting, I guess I will check in ubuntu containers the actual values and have those values set in the interpreter rather than luaLib now that we have structured envs (or if you beat me to it, you seem knowledgeable and enthusiastic about the lua ecosystem ;) ).

Reply:
i'm quite knowledgeable about lua itself, but i'm not actually that in-touch with the ecosystem (for example, i haven't actually used luarocks for any of my projects)

Reply:
> structured envs

i looked this up and got zero results, it seems a lot of new nix features are still underdocumented, could you elaborate?